### PR TITLE
Profile Card #61

### PR DESCRIPTION
**Pull request by g1dra**
Added requested changes. Have some extra logic if user register with only name to generate initials. 
Full name is not required at the moment.

### CODE REVIEW

**Review by jgrossi — CHANGES_REQUESTED** [review]
check the tugboat preview please. not sure if the result is what you imagined

File: app/Http/Controllers/HerdController.php [hunk 1]
```diff
@@ -28,12 +28,19 @@ public function show(string $username)
         $elephpants = $user->elephpants()->orderBy('year', 'desc')->get();
         $userElephpants = $user->elephpantsWithQuantity()->toArray();
 
+        $fullName = explode(' ',$user->name);
+        $initials = substr($fullName[0],0,1);
+        if(isset($fullName[1]))
+        {
+            $initials.=substr($fullName[1],0,1);
+        }
+
```
[the remark this answers is not in the document]
Comment (g1dra) [reply app/Http/Controllers/HerdController.php]: Should be ok now.

**Comment by IgorSantos17** [comment]
Hey,
as I said, there is already an open PR #13,
it even has a good implementation, maybe it would not be better to continue it?

**Comment by g1dra** [comment]
Sure

**Pull request closed without merging**